include/rocksdb/compaction_filter.h

```diff
@@ -332,6 +332,7 @@ class CompactionFilter : public Customizable {
   // and kRemoveAndSkipUntil respectively, and have the same semantics as
   // the corresponding parameters of FilterV2/V3.
   virtual Decision FilterBlobByKey(int /*level*/, const Slice& /*key*/,
```

Comment:
derived compactionfilter will implement FilterBlobByKey to determine if drop this kv according to expire_time. 
since pika store ttl in value，this can skip blobfile random read during compaction.